src/Effects.js

```diff
@@ -682,6 +670,85 @@ module.exports = class Effects {
       }
     };
 
+    this.hearts = {
+      heart: [],
+      size: 50,
+      init: function () {
+        for (let i = 0; i < 20; i++) {
+          this.heart.push({
+            x: randomNumber(10, 390),
+            y: -50,
+            rot: randomNumber(0, 359),
+            speed: p5.random(1,5),
+          });
+        }
+      },
+      update: function () {
+        this.size += randomNumber(-5, 5);
+      },
+      draw: function () {
+        if (this.heart.length < 1) {
+          this.init();
+        }
+        for (let i = 0; i < this.heart.length; i++) {
+          p5.push();
+          const heart = this.heart[i];
```

Comment:
Nit: I find this naming confusing.  I'd prefer `this.hearts` or `this.heartList` for the array.

Reply:
fixed. used heartList.